src/Components/Components/src/ComponentBase.cs

```diff
@@ -37,10 +37,15 @@ public ComponentBase()
         {
             _hasPendingQueuedRender = false;
             _hasNeverRendered = false;
-            BuildRenderTree(builder);
+            RenderCore(builder);
```

Comment:
Instead of making a new `protected virtual` wrapper around `BuildRenderTree`, could we make `Editor<T>` into an `ICascadingValueSupplier`?